src/projectservice/index.js

```diff
@@ -70,9 +72,63 @@ app.post('/createProject', async (req, res) => {
   })
 })
 
+// Endpoint for deleting a project
+app.delete('/deleteProject/:projectID', async (req, res) => {
```

Comment:
As we have discussed in our meetup call, several times the frontend is not storing the projectID, and they will just pass the projectName based on which we will have to extract the projectID.
Please correct this to accept project name either in query param or path param instead of projetId.